### PR TITLE
Winter Coat QOL

### PR DESCRIPTION
**Pull request by beesil**
[qol] 
Makes winter coats not shit.

:cl:
 * rscadd: Gives winter coats pockets, and allows job-specific coats to carry items related to their job in their suit storage.

### CODE REVIEW

File: code/game/objects/items/trader.dm [hunk 1]
```diff
@@ -636,7 +636,7 @@
 										"Try again when you can relate to the intoxicating taste of blood.",
 										"What is a man? A miserable little pile of soft drinks.",
 										"You mortals all look underage to me. Pray tell, can you even manage a bottle?",
-										"Bah. You mock Le Confr�rie des Chevaliers du Tastevin with your plebian visage.",
+										"Bah. You mock Le Confr�rie des Chevaliers du Tastevin with your plebian visage.",
```
Comment (MadmanMartian) [inline code/game/objects/items/trader.dm]: RIP encoding.

Comment (beesil) [reply code/game/objects/items/trader.dm]: fug

Comment (beesil) [reply code/game/objects/items/trader.dm]: I'll try to fix that and the mapping errors tonight, away from my laptop for the next six hours or so.

File: code/game/gamemodes/endgame/xmas/victorianstation.dm [hunk 2]
```diff
@@ -48,56 +48,56 @@
...
 					if(/obj/structure/closet/secure_closet/brig)
-						new /obj/item/clothing/suit/wintercoat/prisoner(S)
-						new /obj/item/clothing/suit/wintercoat/prisoner(S)
+						new /obj/item/clothing/suit/storage/wintercoat/prisoner(S)
+						new /obj/item/clothing/suit/storage/wintercoat/prisoner(S)
 
 					if(/obj/structure/closet/secure_closet/scientist)
-						new /obj/item/clothing/suit/wintercoat/science(S)
-						new /obj/item/clothing/suit/wintercoat/science(S)
+						new /obj/item/clothing/suit/storage/wintercoat/medical/science(S)
```
Comment (MadmanMartian) [inline code/game/gamemodes/endgame/xmas/victorianstation.dm:72]: Might I ask, why the change of making science coats a subtype of medical?

**Review by MadmanMartian — APPROVED** [review]

**Comment by MadmanMartian** [comment]
Mapdiffbot throws a false negative. Looks good to me
